sdk/sdk-core/aar/src/main/java/mil/emp3/core/storage/StorageManager.java

```diff
@@ -897,7 +897,7 @@ public void addOverlays(IMap map, List<IOverlay> overlays, boolean visible) thro
             this.executeTransaction(transactionList);
         } finally {
             lock.unlock();
-            eventManager.generateContainerEvent(ContainerEventEnum.OBJECT_ADDED, map, overlays);
+            eventManager.generateContainerEvent(ContainerEventEnum.OBJECT_ADDED, map, overlays, object);
```

Comment:
Yes, need to review the code and fix it appropriately which could include adding a catch block? Or have a flag that gets set to true at the end of the try block and checked in finally block.